nodejs/json-rpc-relay/integration.tests/mirror.node/web3.clientVersion.test.js

```diff
@@ -83,7 +87,7 @@ describe('[CURL] ClientVersion', () => {
 
         // Assert
         expect(data?.result).toBeDefined();
-        expect(data?.result).toBe("relay/0.17.1");
```

Comment:
This part is quite annoying since i have to replace all around instead of using a variable ..